docs/developer/developing.md

```diff
@@ -0,0 +1,49 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in different terminal windows 
+```
+$ just run-graph-bulider
+```
+
+```
+$ just run-policy-engine
+```
+
+Here is the command to get graph for "stable-4.2" and for "amd64" architecture
```

Comment:
"get graph" -> "get the graph".

And personally, I'd rather have `... to get the stable-4.2 amd64 graph` without the quotes and a reminder that amd64 was an architecture, but I'm fine if you want to keep the quotes and current order.

Reply:
continued in https://github.com/openshift/cincinnati/pull/198#discussion_r375426001